jasypt-spring-boot-starter/src/main/java/com/ulisesbocchio/jasyptspringboot/JasyptSpringBootAutoConfiguration.java

```diff
@@ -1,14 +1,17 @@
 package com.ulisesbocchio.jasyptspringboot;
 
+import com.ulisesbocchio.jasyptspringboot.configuration.EnableEncryptablePropertiesBeanFactoryPostProcessor;
 import com.ulisesbocchio.jasyptspringboot.configuration.EnableEncryptablePropertiesConfiguration;
 
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Import;
 
 /**
  * @author Ulises Bocchio
  */
 @Configuration
+@ConditionalOnMissingBean(EnableEncryptablePropertiesBeanFactoryPostProcessor.class)
```

Comment:
I don't think this is needed here. Are you trying to avoid loading config twice or something?

Reply:
Yes just to avoid loading it for the second time by EnableAutoConfiguration.